messages/results-viewer.md

```diff
@@ -25,3 +25,23 @@
 # summary.detail.breakdown.item
 
 *DRAFT*: %d %s severity
+
+# summary.table.num-column
+
+ #
```

Comment:
`#` is the comment character for MD (equivalent to `//` for C-Like languages), so it took some experimenting to figure out how to have a message consisting of just `#`. Turns out that a leading space will do it.